src/components/vm/vmUsageCard.tsx

```diff
@@ -24,32 +24,41 @@ import type { VM } from '../../types';
 import { Flex, FlexItem } from "@patternfly/react-core/dist/esm/layouts/Flex";
 import { Progress, ProgressVariant } from "@patternfly/react-core/dist/esm/components/Progress";
 
-import { logDebug } from "../../helpers.js";
-
 const _ = cockpit.gettext;
 
 const VmUsageTab = ({ vm } : { vm: VM }) => {
     const memTotal = vm.currentMemory ? vm.currentMemory * 1024 : 0;
     const [memTotalFmt, memTotalUnit] = cockpit.format_bytes(memTotal, { base2: true, separate: true });
-    const rssMem = vm.rssMemory ? vm.rssMemory * 1024 : 0;
-    const memRss = cockpit.format_bytes(rssMem, memTotalUnit, { base2: true, separate: true })[0];
+    let usedMem: number;
+    let usedMemFmt: string;
+    if (vm.state != "running") {
+        usedMem = 0;
+        usedMemFmt = "0";
+    } else if (vm.memoryUsed === undefined) {
+        usedMem = 0;
+        if (vm.hasPollingMemBalloon)
+            usedMemFmt = _("waiting");
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2355-61130920-20251015-073416-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)